src/util/debugging.cpp

```diff
@@ -9,10 +9,19 @@
 
 #include <iostream>
 
+#if !defined(_WIN32) && (defined(__unix__) || defined(__unix) || (defined(__APPLE__) && defined(__MACH__)))
```

Comment:
we have a macro to check if it exists or not: `#ifdef HPX_HAVE_UNISTD_H`
please use that instead.

Reply:
ok. I pasted that from another file, so maybe I should also change that one ...